presto-hive/src/test/java/io/prestosql/plugin/hive/TestBackgroundHiveSplitLoader.java

```diff
@@ -769,7 +769,7 @@ public void testBuildManifestFileIterator()
                 files,
                 directoryLister);
         Optional<Iterator<InternalHiveSplit>> splitIterator = backgroundHiveSplitLoader.buildManifestFileIterator(
-                configuration,
+                new AvroContainerInputFormat(),
```

Comment:
We could use `AVRO.getInputFormat()` to match the above `AVRO.getSerDe()`

Reply:
`AVRO.getInputFormat()` returns string while we need instance here